php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2113,7 +2113,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         // Loop over this instance data
         foreach ($thisData AS $key=>$value) {
             if (!in_array($key, $this->_doubleDataEntryDiffIgnoreColumns)) {
-                if ($otherData[$key] != $value) {
+                if (($otherData[$key] ?? null) !== $value && !empty($value)) {
```

Comment:
Why are you adding a `!empty($value)` check? I think this will cause problems when the value is 0.